senhub.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Actually, this file is a built product, then it should be ignored and never be commited to the main repo. So the challenge is how we can ignore this kind of file?